src/pudl_scrapers/bin/ferc_xbrl.py

```diff
@@ -0,0 +1,131 @@
+"""A command line interface (CLI) to archive data from an RSS feed."""
+import argparse
+import json
+import logging
+import re
+import zipfile
+from pathlib import Path
+
+import feedparser
+import requests
+
+import pudl_scrapers.settings
+from pudl_scrapers.helpers import new_output_dir
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+FERC_RSS_LINK = "https://ecollection.ferc.gov/api/rssfeed"
+
+
+def parse_main():
+    """Process base commands from the CLI."""
+    parser = argparse.ArgumentParser(description="Archive filings from RSS feed")
+    parser.add_argument(
+        "-r", "--rss-path", default=FERC_RSS_LINK, help="Specify path to RSS feed"
+    )
+    parser.add_argument(
+        "-y", "--year", default=2021, type=int, help="Specify single year for filter"
```

Comment:
What's the meaning of the year here? Is it the year that the data pertains to? Or the year in which the data was reported to FERC (which is probably the year after the one that the data pertains to).

By default should this only download a single year? Or should it download all available years starting with 2021, since to create a new archive, we need to download all years?  Otherwise we'll need to run the script many times to create a new archive in subsequent years.